Quadriga/src/main/webapp/WEB-INF/views/settings/addpublicpage.jsp

```diff
@@ -6,24 +6,25 @@
 	href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
 <script>
 //@ sourceURL=filename.js
+	 
 	$(function() {
 		$("#submit1").click(function(){performAction(this)});
 		$("#submit2").click(function(){performAction(this)});
 		$("#submit3").click(function(){performAction(this)});
 	});
-    
+	
 	function setErrorMsg(obj, field, msg) {
 		$(obj).closest('div.publicpageform').find('#' + field + '_err').html(msg);
 	}
 
 	function performAction(obj) {
 		var data = {};
 		var publicpageid = $(obj).closest('div.publicpageform').find('.publicpageid').val();
-		var title = $(obj).closest('div.publicpageform').find('.titleValue').val();
-		var desc = $(obj).closest('div.publicpageform').find('.descValue').val();
-		var order = $(obj).closest('div.publicpageform').find('.orderValue').val();
-		var linkTo = $(obj).closest('div.publicpageform').find('.linkTo').val();
-		var linkText = $(obj).closest('div.publicpageform').find('.linkText').val();
+		var title = $(obj).closest('div.publicpageform').find('#title').val();
```

Comment:
I missed committing this file. I am checking on that and will update pull request